src/components/molecules/ScheduleEvent/ScheduleEvent.tsx

```diff
@@ -27,6 +37,11 @@ import { calcStartPosition } from "components/molecules/Schedule/utils";
 import "./ScheduleEvent.scss";
 
 const ScheduleEventBookmarkClass = "ScheduleEvent__bookmark";
+const ScheduleEventExpandClass = "ScheduleEvent__expand";
+
+const shortEventLength = 10;
+const minLongEventLength = 15;
+const longEventLength = 60;
```

Comment:
All of these should probably go to `settings.ts` with the correct naming pattern.

Reply:
done fd981d9